components/ActivitiesList/ActivitiesList.js

```diff
@@ -59,8 +59,12 @@ export default function ActivitiesList({ handleShowForm }) {
           <Styled.Card
             onClick={handleShowForm}
             style={{ height: "250px", justifyContent: "center" }}
+            // !inline Styles
           >
-            <FiPlus style={{ fontSize: "3rem" }} />
+            <FiPlus
+              style={{ fontSize: "3rem" }}
+              // !inline Styles
+            />
```

Comment:
no inline styles plz ;)

Reply:
yeah well... i hear you and i ignore you :D